packages/now-static-build/src/index.ts

```diff
@@ -229,24 +229,7 @@ export async function build({
         // will ProxyPass any requests to that development server that come in
         // for this builder.
         try {
-          await timeout(
-            new Promise(resolve => {
-              const checkForPort = (data: string) => {
-                // Check the logs for the URL being printed with the port number
-                // (i.e. `http://localhost:47521`).
-                if (data.indexOf(`:${devPort}`) !== -1) {
-                  resolve();
-                }
-              };
-              if (child.stdout) {
-                child.stdout.on('data', checkForPort);
-              }
-              if (child.stderr) {
-                child.stderr.on('data', checkForPort);
-              }
-            }),
-            5 * 60 * 1000
-          );
+          await isPortReachable(devPort, {timeout: 5 * 60 * 1000});
```

Comment:
I don't know why but I expected the library to actually fully wait the timeout in case the host was not reachable. But as you said, if it can't reach the host it directly returns false.